### ISSUE TITLE
What about pipe the response from request() to this.res?

### ISSUE THREAD

**Issue by JasonBoy** [popomore/koa-proxy#43]
can we just pipe the response back to `this.res`?

`this.req.pipe(request(options)).pipe(this.res)` without setting all the response headers and body manually? 
looks like this is much more straight forward.

**Comment by popomore** [comment]
Yes, we can use stream when request the real server, then pipe to response.
